2025-08-17T12:22:52.3284221Z ##[group]Run actions/setup-python@v5
2025-08-17T12:22:52.3284482Z with:
2025-08-17T12:22:52.3284658Z   python-version: 3.11
2025-08-17T12:22:52.3284850Z   cache: pip
2025-08-17T12:22:52.3285061Z   cache-dependency-path: requirements-dev.txt

2025-08-17T12:22:52.3285330Z   check-latest: false
2025-08-17T12:22:52.3285645Z   token: ***
2025-08-17T12:22:52.3285834Z   update-environment: true
2025-08-17T12:22:52.3286055Z   allow-prereleases: false
2025-08-17T12:22:52.3286256Z   freethreaded: false
2025-08-17T12:22:52.3286430Z env:
2025-08-17T12:22:52.3286591Z   PG_USER: postgres
2025-08-17T12:22:52.3286773Z   PG_PASSWORD: pass
2025-08-17T12:22:52.3286948Z   PG_DATABASE: awa
2025-08-17T12:22:52.3287122Z   PG_HOST: localhost
2025-08-17T12:22:52.3287295Z   PG_PORT: 5432
2025-08-17T12:22:52.3287669Z   PG_SYNC_DSN: ***localhost:5432/awa
2025-08-17T12:22:52.3288062Z   PG_ASYNC_DSN: ***localhost:5432/awa
2025-08-17T12:22:52.3288440Z   DATABASE_URL: ***localhost:5432/awa
2025-08-17T12:22:52.3288664Z   DATA_DIR: /tmp/awa-data
2025-08-17T12:22:52.3288885Z   ENABLE_LIVE: 0
2025-08-17T12:22:52.3289123Z   TESTING: 1
2025-08-17T12:22:52.3289309Z ##[endgroup]
2025-08-17T12:22:52.4912423Z ##[group]Installed versions
2025-08-17T12:22:52.5323015Z Successfully set up CPython (3.11.13)
2025-08-17T12:22:52.5323734Z ##[endgroup]
2025-08-17T12:22:52.5665258Z [command]/opt/hostedtoolcache/Python/3.11.13/x64/bin/pip cache dir
2025-08-17T12:22:53.8021235Z /home/runner/.cache/pip
2025-08-17T12:22:53.9093140Z Cache hit for: setup-python-Linux-x64-24.04-Ubuntu-python-3.11.13-pip-dee7c0c28332ec2ebb5d2650fb0d2cfddf025b3b0e96b0c826645ad2cb717961
2025-08-17T12:22:54.9642968Z Received 203535974 of 203535974 (100.0%), 195.5 MBs/sec
2025-08-17T12:22:54.9644048Z Cache Size: ~194 MB (203535974 B)
2025-08-17T12:22:54.9769036Z [command]/usr/bin/tar -xf /home/runner/work/_temp/b5bf184d-0a47-4c6a-87dd-fcd8fd099d6c/cache.tzst -P -C /home/runner/work/AWA-App/AWA-App --use-compress-program unzstd
2025-08-17T12:22:55.3441485Z Cache restored successfully
2025-08-17T12:22:55.3830141Z Cache restored from key: setup-python-Linux-x64-24.04-Ubuntu-python-3.11.13-pip-dee7c0c28332ec2ebb5d2650fb0d2cfddf025b3b0e96b0c826645ad2cb717961
